mpscenes/obstacles/collision_obstacle.py

```diff
@@ -79,3 +79,6 @@ def movable(self):
 
     def update_bullet_position(self, pybullet, **kwargs):
         pass
+
```

Comment:
Can you add the abstractmethod decorator? I think this is needed to enforce that this method is implemented.

Reply:
:+1: